src/Features/Core/EditAndContinue/DebuggingStateChangedEventArgs.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.EditAndContinue
+{
+    internal sealed class DebuggingStateChangedEventArgs
+    {
+        public DebuggingState Before { get; }
```

Comment:
Does anything in this actually use Before, or do we have a future feature that needs it?